controllers/user/UserController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\user;
+
+use humhub\modules\smartVillage\components\AuthBaseController;
+use humhub\modules\smartVillage\controllers\user\GroupController;
+use humhub\modules\user\models\Password;
+use humhub\modules\user\models\Profile;
+use humhub\modules\user\models\User;
+use Yii;
+use yii\web\HttpException;
+use humhub\modules\rest\controllers\auth\AuthController;
+use humhub\modules\rest\models\ConfigureForm;
+use Firebase\JWT\JWT;
+
+
+class UserController extends AuthBaseController
+{
+    /**
+     * Register user without bearer token
+     *
+     * Steps
+     * 1. First, The user register in 'user' table
+     * 2. New user will be automatically adds in default humhub group, which is 'USER' Group whose (id=2)
+     * 3. And finally, User will login with credentials(username and password)
+     *
+     * @return array
+     */
+    public function actionCreate()
```

Comment:
what is the difference between the use controller in `auth/user` and `user`? they seem very similar to me.

Reply:
Hi @donni106 
As far as we know, there is only one usercontroller in the module. Earlier usercontroller was in auth folder but in our last commit we removed the auth folder in controllers and refactor the namespaces.

Or you are pointing out something else?
Thanks!

Reply:
i meant the duplication of files which you can see in the file structure:

<img width="236" alt="Bildschirmfoto 2022-04-29 um 14 35 58" src="https://user-images.githubusercontent.com/1942953/165945442-3f47d6de-beef-4fcb-9905-5210030e29d8.png">

Reply:
Hi @donni106 
Thanks for pointing out. Now the duplication of files are gone on the latest commit we did.